laalaa/apps/advisers/migrations/0009_adding_discrimination_and_education_categories.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-08-30 16:16
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import connection
+
+
+def create_discrimination_and_education_categories(apps, schema_editor):
+    Category = apps.get_model("advisers", "Category")
+    # Hard coding the id values as this is referenced in advisers/fixtures
+    Category.objects.get_or_create(id=14, code="DISC", defaults={"civil": True})
```

Comment:
Does the use of `get_or_create` here mean the environments are not in a consistent state before this migration runs?

Otherwise I would expect you could just run `create` here.

Reply:
I think they will be consistent. However we have this strange scenario where our migration runs on the container entrypoint and have decided to make all migrations idempotent